net-fs/nfs-utils/files/nfs-utils.conf

```diff
@@ -0,0 +1,3 @@
+d /var/lib/nfs/rpc_pipefs
```

Comment:
Is this sufficient for tmpfiles? I always write them with the `-` place holder for default/unused fields.